apps/cyberstorm-storybook/stories/components/Tooltip.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Button, Tooltip } from "@thunderstore/cyberstorm";
+import React from "react";
+
+const meta = {
+  title: "Cyberstorm/Components/Tooltip",
+  component: Tooltip,
+} as ComponentMeta<typeof Tooltip>;
+
+const Template: ComponentStory<typeof Tooltip> = (args) => (
+  <div
+    style={{
+      display: "flex",
+      flexDirection: "column",
+      gap: "0.25em",
+      backgroundColor: "#7e26b6",
```

Comment:
Btw, you can use the variables here too. :P

Reply:
I was harder than I thought it would be. Did change them there 👍